python/cuml/test/test_metrics.py

```diff
@@ -1103,6 +1105,173 @@ def test_pairwise_distances_output_types(input_type, output_type, use_global):
             assert isinstance(S, cp.core.core.ndarray)
 
 
+@pytest.mark.parametrize("metric", PAIRWISE_DISTANCE_SPARSE_METRICS.keys())
+@pytest.mark.parametrize("matrix_size", [(3, 8), (800, 10), (2, 400),
+                                         (500, 40)])
+def test_sparse_pairwise_distances(metric: str, matrix_size):
+    # Test the sparse_pairwise_distance helper function.
+    if metric in ["inner_product", "hellinger"]:
```

Comment:
We definitely want to test these in the Python layer as well. Inner product is going to be a simple `a.dot(b.T)` so we should be able to test that one pretty easily.  

We should also be able to make a naive Hellinger version easily w/ Numpy/Cupy. It's just `euclidean_distance(sqrt(a), sqrt(b)) / sqrt(2)` (though it's important that the input be a probability distribution).

Reply:
I added both naive Inner Product and Hellinger , and the data is normalized to 1 on the rows when using Hellinger.